lib/park.rb

```diff
@@ -0,0 +1,35 @@
+class Park
+  attr_reader :name, :admission_price, :vehicles
+
+  def initialize(name, admission_price)
+    @name = name
+    @admission_price = admission_price
+    @vehicles = []
+  end
+
+  def add_vehicle(vehicle)
+    @vehicles << vehicle
+  end
+
+  def all_passengers
+    @vehicles.flat_map(&:passengers)
```

Comment:
This works, but I would prefer that you not use [proc syntax](https://womanonrails.com/one-line-map-ruby) in Mod1 - it is a lot more advanced than I expect folks to be in these first 6 weeks.